app/services/authService.js

```diff
@@ -115,6 +117,42 @@ class AuthService {
     };
   }
 
+  async sendActivateEmail(userId, userIp) {
+    const oUser = await PortalUserModel.findById(userId).lean();
```

Comment:
Is that the prefix 'o' represents the 'obj'? If yes, in this PR, you have three different styles of naming rules followed.